services/cinema_hall.py

```diff
@@ -0,0 +1,16 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> None:
```

Comment:
The function `create_cinema_hall` is currently annotated to return `None`, but it actually returns a `CinemaHall` instance. Consider changing the return type annotation to `CinemaHall`.